addons/website/static/src/components/dialog/seo.js

```diff
@@ -483,6 +862,36 @@ export class OptimizeSEODialog extends Component {
                 'website_id': this.website.currentWebsite.id,
             },
         });
-        this.website.goToWebsite({path: this.url.replace(this.previousSeoName || this.seoNameDefault, seoContext.seoName)});
+
+        const rpcCalls = [];
+        if (
+            seoContext.brokenLinks &&
```

Comment:
`seoContext.brokenLinks` is always `true` no ?

Reply:
Yep. That's unnecessary